### DISCUSSION TITLE
Make alternate6 - cave dimension (overworld with nether generation)

### DISCUSSION THREAD

**Discussion by ImSpaceLover** [iPortalTeam/ImmersivePortalsMod#1609] [Ideas]

**Comment by LoganDark** [comment]
nether generation has nothing to do with caves

**Comment by ImSpaceLover** [comment]
When you create a buffet world with a cave world generator it looks like nether
![2020-05-24_10 56 11](https://user-images.githubusercontent.com/64267797/82748858-c9470800-9dad-11ea-94cc-f68317c9bedc.png)

![buffet](https://user-images.githubusercontent.com/64267797/82748801-62c1ea00-9dad-11ea-9dc7-79d15e907515.png)

**Comment by LoganDark** [comment]
Huh. I was thinking we could use the same caves as the real overworld does (if this dimension gets implemented of course)
